pepper-apis/docs/specification/src/pepper.yml

```diff
@@ -66,6 +66,8 @@ paths:
     $ref: 'endpoints/admin.studies.participants.yml'
   /admin/studies/{study}/participants-lookup:
     $ref: 'endpoints/admin.studies.participants-lookup.yml'
+  /admin/studies/{study}/participants/{user}:
+    $ref: 'endpoints/admin.studies.participant-lookup-by-guid.yml'
```

Comment:
Can you rename the file to `admin.studies.participant.yml` and move this block to be right below `/admin/studies/{study}/participants`?

Reply:
Fixed